### PR TITLE
tests: Link to instead of copying the default config

### PR DESCRIPTION
**Pull request by fidencio**
By doing this we can ensure that even if the containerd or runtime configuration points to a specific path, changes in the default config will be reflected in the specific path.

### CODE REVIEW

File: integration/nydus/nydus_tests.sh [hunk 1]
```diff
@@ -86,13 +86,13 @@ function config_kata() {
 	sudo mkdir -p /etc/kata-containers
 	if [ -f "$SYSCONFIG_FILE" ]; then
 		need_restore_kata_config=true
-		sudo cp -a "${SYSCONFIG_FILE}" "${kata_config_backup}"
```
Comment (wainersm) [inline integration/nydus/nydus_tests.sh:89]: @fidencio here it saves the file to a backup that is restored later. Either you preserve this backup mechanism or change the logic to switch the link to the default config.

Comment (fidencio) [reply integration/nydus/nydus_tests.sh:89]: Solved, thanks!

Comment (wainersm) [reply integration/nydus/nydus_tests.sh:89]: It is still different: sudo cp -a" ${SYSCONFIG_FILE}" "${kata_config_backup}"

**Review by wainersm — APPROVED** [review]
@fidencio you deserve entering the heaven after changing so many seds

**Comment by fidencio** [comment]
> @fidencio you deserve entering the heaven after changing so many seds

sed++ for helping me to easily change the seds. :-)

File: integration/nydus/nydus_tests.sh (hunk 1 above)
Comment (fidencio) [reply integration/nydus/nydus_tests.sh:89]: Sorry, my bad, fixed now.

**Review by studychao — APPROVED** [review]
LGTM！

**Comment by fidencio** [comment]
Let's drop it for now, as the reason behind doing -- having runtime-rs properly loading ConfigPath -- is already solved.

**Pull request closed without merging**